tinygrad/renderer/cstyle.py

```diff
@@ -58,9 +62,9 @@ def render_load(self, output_dtype, buf_name, buf_dtype, idx, local=False) -> st
       return f"vload_half{'' if output_dtype.count == 1 else str(output_dtype.count)}(0, {buf_name}+{idx})"
     if output_dtype.count > 1:
       out_val = f"*(({self.smem_prefix if local and self.smem_prefix_for_cast else self.buffer_prefix}{self.render_dtype(buf_dtype)}{output_dtype.count}*)({buf_name}+{idx}))"  # noqa: E501
-    else:
-      out_val = f"*({buf_name}+{idx})" if self.uses_ptr_arithmetic else f"{buf_name}[{idx}]"
-    return self.render_cast([out_val], output_dtype) if output_dtype != buf_dtype else out_val
+      return self.render_cast(out_val, output_dtype) if output_dtype != buf_dtype else out_val
```

Comment:
hmm is this diff required?

Reply:
pylint was complaining about this in my pre-commit validation (I assume because I touched the line removing brackets from out_val). I can revert this change and push just my edit if desired.

Reply:
yea it's better if the diff is as small as possible.

Reply:
reverted the unnecessary stuff